src/lexer/nmodl_lexer.hpp

```diff
@@ -80,6 +80,8 @@ class NmodlLexer: public NmodlFlexLexer {
      */
     int lexical_context = 0;
 
+    std::string last_line;
```

Comment:
`pure nitpick` : if you named function as get current line, variable could be cur_line rather than last line? :)

Reply:
yeah.. you're right - couldn't really decide :)